packages/react-components/react-table/src/index.ts

```diff
@@ -1,4 +1,11 @@
-export { useTableFeatures, useTableSelection, useTableSort, createTableColumn } from './hooks';
+export {
+  useTableFeatures,
+  useTableSelection,
+  useTableSort,
+  createTableColumn,
+  useColumnSizing_unstable,
```

Comment:
```suggestion
  useTableColumnSizing_unstable,
```

Let's keep naming consistent to avoid naming conflicts with other packages in the future